simple-instance-v6/instaces-blue.tf

```diff
@@ -0,0 +1,25 @@
+resource "openstack_compute_instance_v2" "instance_blue" {
+  name            = "BLUE Instance"
+  image_id        = data.openstack_images_image_v2.image.id
+  flavor_name     = "m1.tiny"
+  key_pair        = openstack_compute_keypair_v2.kp_adminuser.name
+  security_groups = [openstack_networking_secgroup_v2.sg_ssh.name]
+
+  network {
+    uuid = openstack_networking_network_v2.net_blue.id
+  }
+
+  lifecycle {
+    ignore_changes = [image_id]
+  }
+}
+
+resource "openstack_compute_floatingip_v2" "fip_blue" {
```

Comment:
maybe we should use openstack_networking_floatingip_v2 instead of openstack_compute_floatingip_v2